From: Danny Kukawka <danny.kukawka@web.de>
Subject: remove storage.model property from volumes

Remove storage.model property from volumes. This property make no
sense on a volume device and should be used only within the storage
namespace.

Signed-off-by: Danny Kukawka <danny.kukawka@web.de>
---

 blockdev.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/hald/linux/blockdev.c b/hald/linux/blockdev.c
index d6d2abd..f22af64 100644
--- a/hald/linux/blockdev.c
+++ b/hald/linux/blockdev.c
@@ -1419,7 +1419,6 @@ hotplug_event_begin_add_blockdev (const gchar *sysfs_path, const gchar *device_f
 		hal_device_property_set_string (d, "block.storage_device", hal_device_get_udi (parent));
 
 		/* defaults */
-		hal_device_property_set_string (d, "storage.model", "");
 		hal_device_property_set_string (d, "volume.fstype", "");
 		hal_device_property_set_string (d, "volume.fsusage", "");
 		hal_device_property_set_string (d, "volume.fsversion", "");
